app/policies/permit_application_policy.rb

```diff
@@ -26,7 +26,11 @@ def create?
   end
 
   def update?
-    record.submitter == user
+    record.draft? && record.submitter == user
```

Comment:
we should add this to the specs in particular.  this is some funcitonality to prove that a submitted applicaiton cannot be updated.